### ISSUE TITLE
Anvil Animation Smoke Clipping

### ISSUE THREAD

**Issue by rehatkathuria** [movingparts-io/Pow#21]
The smoke layer from the anvil animation has the top part of it clipped towards the end of the animation.

Tested on Simulator iPhone SE (3rd Generation)

**Comment by robb** [comment]
Looking into it! Do you happen to have the view dimensions at hand? Feel free to send me a screenshot via email robb@movingparts.io if you don't want to post this here.

**Comment by robb** [comment]
Found a way to reproduce:

```swift
import SwiftUI
import Pow

struct ContentView: View {
    @State
    var isOn: Bool = true

    var body: some View {
        VStack {
            GroupBox {
                Toggle("Visible", isOn: $isOn.animation())
            }

            Spacer()

            if isOn {
                Color.blue
                    .frame(width: 200, height: 1)
                    .transition(.movingParts.anvil)
            }

            Spacer()
        }
        .padding()
    }
}

struct ContentView_Previews: PreviewProvider {
    static var previews: some View {
        NavigationStack {
            ContentView()
        }
        .environment(\.colorScheme, .dark)
    }
}
```

**Comment by rehatkathuria** [comment]
Apologies for the slow reply, yesterday was very busy for me! Thank you for addressing/reproducing so quickly!

**Comment by rehatkathuria** [comment]
Will give this a go later and report back, thank you

**Comment by robb** [comment]
Fixed in #23
